src/lib/getClaimIds.js

```diff
@@ -21,3 +29,14 @@ export default function getClaimIds(entity, propId) {
     )
     .filter((c) => c); //filter out 'No value' and 'Unknown'
 }
+
+//checks whether the claim has a qualifier of series ordinal, return true if the first one has it
+export function checkIfClaimsHasSeriesOrdinal(entity, propId) {
+  const claims = entity.claims[propId] || [];
+  if (claims.length) {
+    if (getSeriesOrdinal(claims[0])) {
```

Comment:
are u sure that if the first one has it than it's true for all of them?

Reply:
no, but it's sufficient